src/coprocessor/Client.cc

```diff
@@ -11,8 +11,9 @@ namespace coprocessor
 using namespace std::chrono_literals;
 
 std::vector<copTask> buildCopTasks(
-    kv::Backoffer & bo, kv::Cluster * cluster, std::vector<KeyRange> ranges, Request * cop_req, kv::StoreType store_type)
+    kv::Backoffer & bo, kv::Cluster * cluster, std::vector<KeyRange> ranges, Request * cop_req, kv::StoreType store_type, Logger * log)
```

Comment:
KeyRange shoudl support move construction.

Reply:
will refine in the future